app-text/hunspell/hunspell-1.4.2.recipe

```diff
@@ -73,6 +83,10 @@ INSTALL()
 	make install
 
 	rm $libDir/charset.alias $libDir/*.la
+
+	# remove any existing (but empty) data/locale/*/LC_MESSAGES directory
+	rmdir $dataDir/locale/*/LC_MESSAGES $dataDir/locale/* || true
```

Comment:
It is not empty. It has some comments and then 47 lines of aliases for locales.
BTW, the last comment is «**`# Packages using this file: hunspell`**». 
I guess we should keep the **`locale.alias`** file, at least until we add a hunspell dictionary and test hunspell without the **`locale.alias`** file.